src/util/options.rs

```diff
@@ -672,88 +700,85 @@ mod gc_trigger_tests {
 // Currently we allow all the options to be set by env var for the sake of convenience.
 // At some point, we may disallow this and all the options can only be set by command line.
 options! {
-    // The plan to use.
+    #[doc = "The plan to use."]
```

Comment:
Like the other PR, we should be able to use `///` here, too.

Reply:
`///` may not work here. We can use `#[doc]` as we capture the pattern `$(#[$outer:meta])*` in the macro. We should be able to use the same pattern to capture things like `#[cfg(feature = "a")]`. This seems to be a common way to add attributes for items that are generated by a macro.

Reply:
Just to clarify, I was also using `#[doc]` to pass attributes (including docs) to macros in the previous PR.

Reply:
Actually it works.  I changed it to 

```rust
    /// The plan to use. Blah blah. (to see if `///` works)
    plan:                  PlanSelector         [env_var: true, command_line: true] [always_valid] = PlanSelector::GenImmix,
```

Type `cargo doc` and the generated HTML contains the part in `///`.

FYI: https://stackoverflow.com/questions/33999341/generating-documentation-in-macros

Reply:
Yeah. Interesting. It works. Rust did not allow `///` in `macro_rules` and that was why we have `//` in the `options!` macro. Probably Rust started to support this at some point. This is great. I will make these changes.

Reply:
Oh I see. `#[$outer:meta]*` matches `///`.